<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="7" rowspan="3">
    <widget class="QSlider" name="S_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="5" rowspan="3">
    <widget class="QSlider" name="S_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="8">
    <widget class="QPushButton" name="Browse_Button">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="PlotWidget" name="Filtered_view"/>
   </item>
   <item row="1" column="4" rowspan="3">
    <widget class="QSlider" name="S_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <widget class="PlotWidget" name="Signal_freq"/>
   </item>
   <item row="1" column="2" rowspan="3">
    <widget class="QSlider" name="S_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="9">
    <widget class="PlotWidget" name="Filtered_freq"/>
   </item>
   <item row="1" column="3" rowspan="3">
    <widget class="QSlider" name="S_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="PlotWidget" name="Signal_view"/>
   </item>
   <item row="1" column="1" rowspan="3">
    <widget class="QSlider" name="S_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QSlider" name="S_1">
     <property name="minimum">
      <number>-30</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="6" rowspan="3">
    <widget class="QSlider" name="S_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="9">
    <widget class="QPushButton" name="Play_button">
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
   <item row="6" column="8" colspan="2">
    <widget class="QPushButton" name="live_sound">
     <property name="text">
      <string>Live Sound</string>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Signal in time domain</string>
     </property>
    </widget>
   </item>
   <item row="4" column="8">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Signal in Frequency Domain</string>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Filtered Signal in time domain</string>
     </property>
    </widget>
   </item>
   <item row="4" column="9">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Filtered Signal in Frequency domain</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>0 dB</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>0 dB</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>0 dB</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>0 dB</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>0 dB</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>0 dB</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>0 dB</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>0 dB</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
